code/src/App.js

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import { Age } from 'components/Age';
+import { Gender } from 'components/Gender';
+import { Electricity } from 'components/Electricity';
+import { Result } from 'components/Result';
 
 export const App = () => {
+  const [step, setStep] = useState(1);
+  const [age, setAge] = useState('');
+  const [gender, setGender] = useState('');
+  const [electricity, setElectricity] = useState('');
+
+  const handleStepIncrease = () => {
```

Comment:
Could be in it's own component, but as it is very short code it almost works better here as it would be more work creating a whole new component for it.